safe/impact_functions/generic/continuous_hazard_population/metadata_definitions.py

```diff
@@ -100,7 +102,7 @@ def as_dict():
             },
             # Configurable parameters
             'parameters': OrderedDict([
-                ('Categorical thresholds', [0.34, 0.67, 1]),
+                ('Categorical thresholds', categorical_thresholds()),
```

Comment:
This should be explicitly named (i.e. should have flood in the name). I am also confused as to why this parameter is not called ``continuous_flood_threshold`` and its title ``tr('Flood threshohlds'))``